config/notifiers.go

```diff
@@ -763,6 +764,9 @@ func (c *PushoverConfig) UnmarshalYAML(unmarshal func(interface{}) error) error
 	if c.Token != "" && c.TokenFile != "" {
 		return errors.New("at most one of token & token_file must be configured")
 	}
+	if c.HTML && c.Monospace {
+		return errors.New("monospace may not be used if html is used, and vice versa")
```

Comment:
Keeps it consistent will standard error message text:

```suggestion
		return errors.New("at most one of monospace & html must be configured")
```